backbone.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>If you'd prefer to do this more rigorously we could try to setup a <code>unittest</code>  or <code>pytest</code>  framework. For now, LGTM.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yycho0108/ai604-video-object-pose/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>